data/sheragi/the emerald sword.txt

```diff
@@ -0,0 +1,854 @@
+			`	But before you can initiate the approach to the ship, a strange feeling washes over you. First, a feeling of being watched by something far greater. Then, a void in your chest, a deep apprehension you can't shake. A thought forms in your mind, but it's not your own:`
+			`	"You have arrived, human. This was the Sheragi's last stand before their extinction. You would do well not to disturb this place of remembrance."`
+			`	A strong shiver washes over you as the thoughts of "extinction" and "remembrance" are punctuated by an indescribable feeling. As if there is a cold in your bones.`
+			choice
+				`	"Who are you?"`
+					goto who
+				`	"What... What is this feeling?"`
+			`	There is a pause. The feeling of being watched becomes intense, as if you're being studied intently.`
+			`	"A human would call it... dread. Regret. Loss. You are standing in the graveyard of a lost civilization.`
+			label who
+			`	"I am the guardian of this site. I am also the one who guided you towards the buried city, so your species would bear witness and take heed of the lessons of ones who have long succumbed to their own folly. I did not intend for you to reach this far.`
+			`	"Here is where the Sheragi came to an end. The tools that drove them to extinction are long gone. We witnessed as they cast those tools into the heart of this star."`
+			choice
+				`	"You mean this ship? Its weapons systems seem to be intact."`
+					goto cannon
+				`	"What tools?"`
+			`	"Ones capable of crippling an entire world's worth of life."`
+				goto next
+			label cannon
+			`	"That ship is just a relic. The weapon that perished the Sheragi was indiscriminate. An irrational decision they made themselves."`
+			label next
+			`	As that is said, you're overcome by glimpses of memories that are not yours, not clear enough for you to identify what exactly is happening, but you can feel the sadness in them.`
+			`	The voice continues: "I know what you seek here. The trinket you covet is of little relevance, even for your own, human standards. But if you wish to possess it, then take it. Perhaps the sight of that vessel will remind the living races of those who have come and gone... and how, despite their great accomplishments, their decision to make war reduced them to graveyards, relics, and memories."`
+			`	The voice, and the overwhelming emotions that came alongside it, disappear as mysteriously as they came. Desyo, who witnessed you speaking into thin air, asks whether you feel okay. It seems like no one else heard it.`
```

Comment:
**ERROR**
Just about this entire conversation in general: Are you supposed to be speaking to a Drak? If so, why is this one not visible? I could just be missing something in the coding, but I remember the other time you interact with Drak, there is one in the system, and I don't think there's one here. If not, then you're good. Just checking.

Reply:
Drak have the ability to cloak themselves indefinitely, as seen in Nenia and Ildaria, so it isn't entirely wrong. I decided to omit this one, so that only the player can communicate with it.